lib/ex_format.ex

```diff
@@ -1,1371 +1,23 @@
 import Kernel, except: [to_string: 1]
+import ExFormat.Formatter, only: [to_string_with_comments: 1]
+import ExFormat.State
+import ExFormat.AST
+import ExFormat.Comments
+import ExFormat.Lines
```

Comment:
Since this module is fairly small, don't import anything. Use 

```elixir
alias ExFormat.{
  Formatter,
  State,
  ...,
}
```

and use qualified calls. `import` is considered a last resource and should be used with a lot of care.